src/runtime-rs/crates/resource/src/share_fs/utils.rs

```diff
@@ -41,6 +41,14 @@ pub(crate) fn share_to_guest(
 
     Ok(do_get_guest_path(target, cid, is_volume))
 }
+// Shared path handling:
+// 1. create two directories for each sandbox:
+// -. /run/kata-containers/shared/sandboxes/$sbx_id/rw/, a host/guest shared directory which is rw
+// -. /run/kata-containers/shared/sandboxes/$sbx_id/ro/, a host/guest shared directory (virtiofs source dir) which is ro
+//
+// 2. /run/kata-containers/shared/sandboxes/$sbx_id/rw/ is bind mounted readonly to /run/kata-containers/shared/sandboxes/$sbx_id/ro/, so guest cannot modify it
+//
+// 3. host-guest shared files/directories are mounted one-level under /run/kata-containers/shared/sandboxes/$sbx_id/rw/ and thus present to guest at one level under /run/kata-containers/shared/sandboxes/$sbx_id/shared/
```

Comment:
Is `/run/kata-containers/shared/sandboxes/$sbx_id/shared/` should be `/run/kata-containers/shared/sandboxes/$sbx_id/ro/`?